src/Bootstrap/src/Autoconfig/SteeltoeAssemblies.cs

```diff
@@ -6,35 +6,28 @@ namespace Steeltoe.Bootstrap.Autoconfig;
-    public const string SteeltoeExtensionsConfigurationCloudFoundryBase = "Steeltoe.Extensions.Configuration.CloudFoundryBase";
-    public const string SteeltoeExtensionsConfigurationCloudFoundryCore = "Steeltoe.Extensions.Configuration.CloudFoundryCore";
-    public const string SteeltoeExtensionsConfigurationConfigServerBase = "Steeltoe.Extensions.Configuration.ConfigServerBase";
-    public const string SteeltoeExtensionsConfigurationConfigServerCore = "Steeltoe.Extensions.Configuration.ConfigServerCore";
-    public const string SteeltoeExtensionsConfigurationKubernetesBase = "Steeltoe.Extensions.Configuration.KubernetesBase";
-    public const string SteeltoeExtensionsConfigurationKubernetesCore = "Steeltoe.Extensions.Configuration.KubernetesCore";
-    public const string SteeltoeExtensionsConfigurationRandomValueBase = "Steeltoe.Extensions.Configuration.RandomValueBase";
-    public const string SteeltoeExtensionsConfigurationPlaceholderBase = "Steeltoe.Extensions.Configuration.PlaceholderBase";
-    public const string SteeltoeExtensionsConfigurationPlaceholderCore = "Steeltoe.Extensions.Configuration.PlaceholderCore";
-    public const string SteeltoeConnectorEf6Core = "Steeltoe.Connector.EF6Core";
-    public const string SteeltoeConnectorEfCore = "Steeltoe.Connector.EFCore";
-    public const string SteeltoeConnectorConnectorCore = "Steeltoe.Connector.ConnectorCore";
-    public const string SteeltoeDiscoveryClientBase = "Steeltoe.Discovery.ClientBase";
-    public const string SteeltoeDiscoveryClientCore = "Steeltoe.Discovery.ClientCore";
-    public const string SteeltoeExtensionsLoggingDynamicSerilogCore = "Steeltoe.Extensions.Logging.DynamicSerilogCore";
+    public const string SteeltoeCircuitBreakerHystrixMetricsEvents = "Steeltoe.CircuitBreaker.Hystrix.MetricsEvents";
+    public const string SteeltoeCircuitBreakerHystrixMetricsStream = "Steeltoe.CircuitBreaker.Hystrix.MetricsStream";
+    public const string SteeltoeCircuitBreakerHystrix = "Steeltoe.CircuitBreaker.Hystrix";
+    public const string SteeltoeExtensionsConfigurationCloudFoundry = "Steeltoe.Extensions.Configuration.CloudFoundry";
+    public const string SteeltoeExtensionsConfigurationConfigServer = "Steeltoe.Extensions.Configuration.ConfigServer";
+    public const string SteeltoeExtensionsConfigurationKubernetes = "Steeltoe.Extensions.Configuration.Kubernetes";
+    public const string SteeltoeExtensionsConfigurationRandomValue = "Steeltoe.Extensions.Configuration.RandomValue";
+    public const string SteeltoeExtensionsConfigurationPlaceholder = "Steeltoe.Extensions.Configuration.Placeholder";
+    public const string SteeltoeConnectorEntityFramework6 = "Steeltoe.Connector.EntityFramework6";
+    public const string SteeltoeConnectorEntityFrameworkCore = "Steeltoe.Connector.EntityFrameworkCore";
+    public const string SteeltoeConnectorConnector = "Steeltoe.Connector";
```

Comment:
I suspect this constant should be renamed to ~SteeltoeConnectorConnector~ SteeltoeConnector, in alignment with the other renames.